micro-rdk/src/common/conn/viam.rs

```diff
@@ -643,6 +656,7 @@ where
                     })
                     .ok()
             } else {
+                log::info!("falling back to stored TLS certificate");
```

Comment:
It says what but not why. Maybe `log::info!("Failed to obtain certificates from {}, will attempt to load any stored certificates", self.app_client.uri()")` or similar. You could decide against logging the app address if you want, since otherwise you'd need to do the work to expose the URI held in the inner GrpcClient.